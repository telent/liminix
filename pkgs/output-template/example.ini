wpa_passphrase={{ secret "colours/black"  }}
think = {{ string.format("%q", secret("colours/blue")) }}
