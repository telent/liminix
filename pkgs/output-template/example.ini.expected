wpa_passphrase=000000
think = "0000ff"
